compiler/ballerina-backend-jvm-old/src/main/ballerina/compiler_backend_jvm/jvm_method_gen.bal

```diff
@@ -1543,7 +1543,12 @@ function generateLambdaForPackageInits(jvm:ClassWriter cw, bir:Package pkg,
         mv.visitInsn(ICONST_0);
         mv.visitInsn(AALOAD);
         mv.visitTypeInsn(CHECKCAST, STRAND);
-        mv.visitMethodInsn(INVOKESTATIC, initClass, initFuncName, io:sprintf("(L%s;)V", STRAND), false);
+
+        if (pkg.org.value == BALLERINA && pkg.name.value == BUILT_IN_PACKAGE_NAME) {
```

Comment:
Do we need to special case builtin?

Reply:
may be error is not there yet?

Reply:
Yeah, the error type is null when we are setting the return type for the lang.annotations module. Will update that on the compiler side to make it `returns any` instead of `returns ()` as discussed offline with @lankavitharana to avoid special casing in the backend.